client/signup/config/flows-pure.js

```diff
@@ -112,7 +112,7 @@ export function generateFlows( {
 		},
 		{
 			name: 'onboarding',
-			steps: [ 'user', 'domains', 'plans' ],
+			steps: [ 'user', 'domains', 'emails', 'plans' ],
```

Comment:
I mentioned this verbally yesterday, but I think we should feature flag this ASAP. I've suggested a flag name below, but we don't have to use this one.

For now, let's keep the feature disabled in all environments until we have it sufficiently wired-in/working. That means we shouldn't add it to `config/development.json` quite yet.

```suggestion
			import { isEnabled } from '@automattic/calypso-config'; // add at top of file

			steps: isEnabled( 'signup/professional-email-step' ) ? [ 'user', 'domains', 'emails', 'plans' ] : [ 'user', 'domains', 'plans' ],
```

Reply:
Thank you, I must admit that my intention was to keep working on it, so I delayed it, like a few things more in the code that are now solved.